docs/sources/reference/run.md

```diff
@@ -348,6 +349,16 @@ with time quantum set as 100 milliseconds, containers C0, C1, and C2 will run
 for full-time quantum, and container C3 will run for half-time quantum i.e 50
 milliseconds.
 
+The flag '--cpu-quota' with value 0 indicates that the running container 
+can use the full 100000 (default) cpu cfs (cfs is Completely Fair Scheduler 
+which is default scheduler used by Linux kernel) period. Howerver, this value 
+can be modified to run a container with a different usage of cpu cfs period.
+
+E.g., If we start a container with default values (`--cpu-quota` = 0)
+the container can use 100% cpu resource if it is need and if you start with
+('cpu-quota' = 50000), the container only can use 50% of the cpu resource
+at maximum.
+
```

Comment:
To limit the CPU available to a running container, use the '--cpu-quota' flag. The default of '0' allows the container to take the full 100000 CPU CFS (Complete Fair Scheduler) or 100%.  The CFS is the Linux kernel's scheduler.  Set this value to `50000` to limit the container to 50% of the CPU resource.